src/Areas/KeyVault/Commands/Certificate/CertificateCreateCommand.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.KeyVault.Options;
+using AzureMcp.Areas.KeyVault.Options.Certificate;
+using AzureMcp.Areas.KeyVault.Services;
+using AzureMcp.Commands.KeyVault;
+using AzureMcp.Commands.Subscription;
+using AzureMcp.Services.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.KeyVault.Commands.Certificate;
+
+public sealed class CertificateCreateCommand(ILogger<CertificateCreateCommand> logger) : SubscriptionCommand<CertificateCreateOptions>
+{
+    private const string CommandTitle = "Create Key Vault Certificate";
+    private readonly ILogger<CertificateCreateCommand> _logger = logger;
+    private readonly Option<string> _vaultOption = KeyVaultOptionDefinitions.VaultName;
+    private readonly Option<string> _certificateOption = KeyVaultOptionDefinitions.CertificateName;
+
+    public override string Name => "create";
+
+    public override string Title => CommandTitle;
+
+    public override string Description =>
```

Comment:
Did you try out the confidence score with the test prompt? If so, can you include that data in the PR description?